fetch/api/resources/utils.js

```diff
@@ -44,6 +44,11 @@ function checkRequest(request, ExpectedValuesDict) {
   }
 }
 
+function encode_utf8(str)
+{
+    return unescape(encodeURIComponent(str));
```

Comment:
Or use TextEncoder if available and fallback otherwise.
That is what encrypted-media/util/utf8.js is doing